packages/tools/kadena-cli/src/devnet/utils/docker.ts

```diff
@@ -141,3 +141,13 @@ export function updateDevnet(version?: string): void {
   execSync(`docker pull ${image}`);
   console.log(chalk.green(`Updated ${image}`));
 }
+
+export function containerIsRunning(name: string): boolean {
+  const runningContainers = execSync(
+    'docker ps --filter ancestor=kadena/devnet --format "{{.Names}}"',
+  )
+    .toString()
+    .trim()
+    .split('\n');
+  return runningContainers.includes(name);
+}
```

Comment:
When using `ancestor=kadena/devnet`, argument `name` becomes obsolete and you can check whether the amount if lines returned by `docker ps .... ` > 0, right?

Reply:
https://github.com/kadena-community/kadena.js/pull/1469#discussion_r1457389453

Reply:
The reason for verifying the name is that we can make sure the chosen devnet config is indeed the one running. This way we not only validate that a devnet is indeed running but also that the devnet configuration chosen is the correct one. This is useful because we can therefore use the port in that config without the need for an additional promt